changelog.d/12709.removal

```diff
@@ -0,0 +1 @@
+Require a body in POST requests to `/rooms/{roomId}/receipt/{receiptType}/{eventId}`, as required by the [Matrix specification](https://spec.matrix.org/v1.2/client-server-api/#post_matrixclientv3roomsroomidreceiptreceipttypeeventid). This breaks compatibility with Element Android 1.2.0 and earlier: users of those clients will be unable to send read receipts. See [#12709](https://github.com/matrix-org/synapse/pull/12709).
```

Comment:
Doesn't a link to the PR get appended automatically by towncrier?

Reply:
Sorry, this was meant to be a link to the underlying issue, #10534, which has the additional context. But even with that said, I suppose the issue will be discoverable from this PR.